src/platform/forms-system/src/js/containers/FormPage.jsx

```diff
@@ -6,6 +6,7 @@ import get from '../../../../utilities/data/get';
 import set from '../../../../utilities/data/set';
 import classNames from 'classnames';
 
+import { isLoggedIn } from 'platform/user/selectors';
```

Comment:
(I forgot to add this before I submitted the review)

I realize that in here we have direct access to the redux store, so it's convenient to get `isLoggedIn` and pass it as an argument. However, to get that value where it needs to go we're modifying 6 more functions across 3 different files, and this is only a draft so the final implementation would likely have more. This kind of approach easily gets cumbersome, so I wonder if it would be easier to instead get this value from the store directly when and where we need it rather than passing it around so much just so a helper in a grandchild component can make use of it.

Reply:
So accessing the store gets a bit dicey because of the way that our application works. Technically the only place we can really access the store is inside the application itself, because the redux store gets instantiated when the application calls `startApp`. There are intermediate functions that have access to the store object as well, but I can't export the store out of those files because you can only exports `consts`. Maybe there is a way that I haven't thought of to accomplish this though

Reply:
`startApp` does return the store though, so would something like this work?

https://github.com/department-of-veterans-affairs/vets-website/blob/90b36e43ace72cd8561d668e342272a5d35aaa3f/src/applications/facility-locator/facility-locator-entry.jsx#L10-L17

https://github.com/department-of-veterans-affairs/vets-website/blob/90b36e43ace72cd8561d668e342272a5d35aaa3f/src/applications/facility-locator/api/LocatorApi.js#L28

---

I'm not sure I understand what you mean here with the export problem, can you show me an example?
> There are intermediate functions that have access to the store object as well, but I can't export the store out of those files because you can only exports `consts`

Reply:
Yeah, so this is fine as long as you want to access the store within the application code, but if you were to attempt to access the store from `src/platform` where would you import the store from? Getting the store from `facility-locator` only works when the user is using that app, but if they are on the `lgy/coe` app for instance, startApp won't have been called for `facility-locator` so that store object won't be useful

Reply:
So after realizing that I looked into getting the store from `startApp`
https://github.com/department-of-veterans-affairs/vets-website/blob/fb0d08cac689665d24a90c4c137debb52e40ddbe/src/platform/startup/index.js#L37
by creating a variable outside of startApp and assigning the created store to that variable, but it's bad practice to export a variable if it's not a const